### PR TITLE
[Merged by Bors] - Update TF To 2.6.0

### PR DESCRIPTION
**Pull request by xueshe**

### CODE REVIEW

**Comment by shiyingjin** [comment]
I think the modification is incomplete

**Comment by NaNAGISaSA** [comment]
@shiyingjin Please review this PR, especially the struct of bazel project files.

File: ci/azure-pipelines/jobs/bazel-build-serving-tensorrt.yml [hunk 1]
```diff
@@ -23,7 +23,8 @@ jobs:
                     -c opt \
                     --disk_cache="$BAZEL_BUILD_CACHE" \
                     --action_env=LIBRARY_PATH=/usr/local/cuda-11.0/lib64/stubs \
-                    --incompatible_use_specific_tool_files=false
+                    --incompatible_use_specific_tool_files=false \
+                    --copt="-Ithird_party/cuda/include"
```
Comment (shiyingjin) [inline ci/azure-pipelines/jobs/bazel-build-serving-tensorrt.yml:27]: Need "-Ithird_party/cuda/include" ? See adlik_serving/runtime/tensorrt/model/BUILD

Comment (xueshe) [reply ci/azure-pipelines/jobs/bazel-build-serving-tensorrt.yml]: Without this config, bazel cannot build it successfully. Only adding in "adlik_serving/runtime/tensorrt/model/BUILD" does not work.

**Comment by shiyingjin** [comment]
bors retry

**Comment by shiyingjin** [comment]
bors retry

**Comment by shiyingjin** [comment]
bors retry

**Comment by NaNAGISaSA** [comment]
bors retry

**Pull request closed without merging**